truth-sets/GRCh37/eichler/README.md

```diff
@@ -1,3 +1,4 @@
 This dataset is the set of de novos in autism spectrum disorder from [Iossifov et al, 2014](https://www.nature.com/nature/journal/v515/n7526/full/nature13908.html).
 Eichler's pathogenic de novos from this set are not necessarily pathogenic absolutely, but are of strong genotype quality, present only in the proband, or experimentally validated in some way.
 The benigns are everything else.
+It should also be noted that while we call the set "Eichler," due credit should be given to the four corresponding authors whose labs contributed: Jay Shendure, Evan E. Eichler, Matthew W. State, and Michael Wigler, the last of which is the the last author.
```

Comment:
if you're going to add this, can you add the citations?

Reply:
The citation is there.  What do you mean?